src/docs/Dependency injection.md

```diff
@@ -23,22 +27,12 @@ system.ActorOf(propsResolver.Create<TypedWorker>(), "Worker2");
 
 ```
 
-Creating child Actors from inside an Actor can be done using the DI() extension method.
```

Comment:
Any specific reason why you removed this part? As far as I know if we remove this, this won't be described anywhere. And I think we should.